purchase_order_reorder_lines/purchase.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Author: Alexandre Fayolle
+#    Copyright 2013 Camptocamp SA
+#    Author: Damien Crier
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+from openerp import models, api, fields
+
+
+class purchase_order_line(models.Model):
+    _inherit = 'purchase.order.line'
+
+    sequence = fields.Integer(default=10,
```

Comment:
It would be good to have a _default method that query the order to put the previous sequence plus 1, instead of assigning an static value. In this case, when you have more than 10 lines already introduced, you reorder, and then insert a new line, this line will be intercalated between the others.